docs/releases/5.0.txt

```diff
@@ -432,8 +432,8 @@ Tests
 
 * :class:`~django.test.AsyncClient` now supports the ``follow`` parameter.
 
-* The new :option:`test --durations` option allows showing the duration of the
-  slowest tests on Python 3.12+.
+* :class:`~django.test.runner.DiscoverRunner` now allows showing the duration
+  of the slowest tests using the :option:`test --durations` option.
```

Comment:
Why is this change dropping the `on Python 3.12+` clarification? I think it's necessary to keep it, otherwise this release note can cause considerable confusion (I spent a while debugging why my test runner did not have it! until I checked the source).

Reply:
I thought the grammar wasn't quite right (perhaps implying the version of Python has something to do with the speed of tests rather than indicating the availability of the --durations option) and it seemed more detail than necessary. If you click on the option to learn more, there is a prominent warning. I suppose I shall add it back.

Reply:
Thanks for the clarification, I see what you mean now. I still think that a small clarification in the release note is worth the effort. What about (needs proper wrapping of course):
```suggestion
  of the slowest tests using the :option:`test --durations` option (available on Python 3.12+).
```